### PR TITLE
Sayakd915/meetingplanpage

### LINKED ISSUE #4: meeting plan page
### Description

<b>create seperate branch named-> <yourname>-page or component name you are working on</b>

use dropdown box and for now add placeholders we will use backend to populate the data.  (<b>Note: For time use TimePicker widget to select the time</b>)
Proper validation should be done for each field.

### Preview

image

### PR DESCRIPTION
**Pull request by Sayakd915**
commit : Meeting Plan Page added

https://github.com/user-attachments/assets/181a7759-a50f-40e4-8e6c-2a601e4b30cf

fix #4

### CODE REVIEW

File: lib/Themes/themes.dart
Comment (SHREYANSHSK) [inline lib/Themes/themes.dart]: don't import raleway font explicitly, GoogleFonts.raleway to use raleway font, add the lines that you have deleted fot TextStyle

File: lib/app/modules/meetings/views/meetings_view.dart
Comment (SHREYANSHSK) [inline lib/app/modules/meetings/views/meetings_view.dart]: add all the variable and list to  controller file and remove unnecessary comments

File: lib/app/modules/meetings/views/meetings_view.dart
Comment (SHREYANSHSK) [inline lib/app/modules/meetings/views/meetings_view.dart]: instead of defining TextStyle to individual text, you can use the TextStyle that has already been set up (That you have removed from themes.dart)

File: lib/app/modules/meetings/views/meetings_view.dart
Comment (SHREYANSHSK) [inline lib/app/modules/meetings/views/meetings_view.dart]: use *Get.theme.ColorLevel* instead of giving individual colors

File: lib/app/modules/meetings/views/meetings_view.dart
Comment (SHREYANSHSK) [inline lib/app/modules/meetings/views/meetings_view.dart:40]: here also use *Get.theme.colorLevel*  the colours and theme has already been set up in themes.dart refer to that, and do the same wherever necessary

File: lib/app/modules/meetings/views/meetings_view.dart
Comment (SHREYANSHSK) [inline lib/app/modules/meetings/views/meetings_view.dart]: remove bottom Navigattion bar for now we will add this afterwards

File: lib/app/modules/meetings/views/meetings_view.dart
Comment (SHREYANSHSK) [inline lib/app/modules/meetings/views/meetings_view.dart]: make a separate file for this widget and place it in component folder

File: lib/app/modules/meetings/views/meetings_view.dart
Comment (SHREYANSHSK) [inline lib/app/modules/meetings/views/meetings_view.dart]: do the same as above in this and wherever necessary

**Review by SHREYANSHSK — CHANGES_REQUESTED** [review]
remove the downloaded raleway font and use GoogleFonts for that, 

also the lot of mismatch in Your design is their as compared to figma file try to make the design as accurate as possible refer to figma file if needed

check the gradient part as well

File: lib/app/modules/meetings/views/meetings_view.dart
Comment (SHREYANSHSK) [inline lib/app/modules/meetings/views/meetings_view.dart:22]: use get.theme for theme colour

File: lib/app/modules/meetings/controllers/meetings_controller.dart
Comment (SHREYANSHSK) [inline lib/app/modules/meetings/controllers/meetings_controller.dart:3]: where are the functions that are automatically created by get_cli like on ready function , on dispose function and others in controller file
if you have removed that make sure you dont do that from next time as it plays very important role to prevent memory leaks
just add these functions again

```
  @override
  void onInit() {
    super.onInit();
  }

  @override
  void onReady() {
    super.onReady();
  }

  @override
  void onClose() {
    super.onClose();
  }

  void increment() => count.value++;
  
```

**Review by SHREYANSHSK — CHANGES_REQUESTED** [review]
all things looks good for now just do some  minor changes

<li>remove devtools_options.yaml file if it is of no use </li>
<li>puts assets folder outside of lib folder that is in root and put your image in  assets/images </li>
<li>put components folder outside of lib folder that is in root</li>
<li>upload latest preview</li>

After this i will probably merge the pr

File: lib/app/modules/meetings/controllers/meetings_controller.dart
Comment (Sayakd915) [reply lib/app/modules/meetings/controllers/meetings_controller.dart:3]: I haven't removed these functions. These just weren't there when I was working on the controller

**Comment by Sayakd915** [comment]
commit -m "Meeting Plan Page Edited"

fix #4 


https://github.com/user-attachments/assets/1c423092-9726-4c72-b142-72b2fbafe65b

**Review by SHREYANSHSK — APPROVED** [review]
looks good i will merge it after resolving conflicts

**Pull request merged**
